ci/sawtooth-build-debs

```diff
@@ -83,7 +84,8 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
     unzip \
     wget \
  && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+ && rm -rf /var/lib/apt/lists/* \
+ && pip3 install mock --upgrade
```

Comment:
Can we use the python3-mock package instead (if this exists...)?

Reply:
I don't think there is one. The reason for this install is because in Python 3.3+ mock is a standard library and the import is slightly different. Something like `import unittest.mock` instead of `import mock`